install/jms/bin/ts.jte.jdk11

```diff
@@ -205,39 +204,39 @@ tslib.name=jmstck
 
 #########################################################################
 # platform.mode is used by to specify runtime environment for testing
-# which can be either javaEE or standalone. For JMS TCK this setting
+# which can be either javaEE or standalone. For Jakarta Messaging TCK this setting
```

Comment:
same as [this line 213](https://github.com/eclipse-ee4j/jakartaee-tck/pull/415/files#diff-34b05b6c2845c35e2d7a9f4be79b0216R213)

Reply:
@gabrielbussolo is your comment about line 213 addressed?